WORKSPACE

```diff
@@ -0,0 +1,71 @@
+'''This file lists and imports all external dependencies needed to build Oppia Android.
+'''
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:jvm.bzl", "jvm_maven_import_external")
+
+# Android SDK configuration. For more details, see:
+# https://docs.bazel.build/versions/master/be/android.html#android_sdk_repository
+android_sdk_repository(
+    name = "androidsdk",
+    api_level = 28,
+    build_tools_version = "28.0.2",
+)
+
+# Add support for JVM rules: https://github.com/bazelbuild/rules_jvm_external
+RULES_JVM_EXTERNAL_TAG = "2.9"
+RULES_JVM_EXTERNAL_SHA = "e5b97a31a3e8feed91636f42e19b11c49487b85e5de2f387c999ea14d77c7f45"
+http_archive(
+    name = "rules_jvm_external",
+    strip_prefix = "rules_jvm_external-%s" % RULES_JVM_EXTERNAL_TAG,
+    sha256 = RULES_JVM_EXTERNAL_SHA,
+    url = "https://github.com/bazelbuild/rules_jvm_external/archive/%s.zip" % RULES_JVM_EXTERNAL_TAG,
+)
+
+load("@rules_jvm_external//:defs.bzl", "maven_install")
+
+# Add support for Kotlin: https://github.com/bazelbuild/rules_kotlin.
+RULES_KOTLIN_VERSION = "legacy-1.4.0-rcx-oppia-exclusive-rc01"
+RULES_KOTLIN_SHA = "600f3d916eda5531dd70614ec96dc92b4ac24da0e1d815eb94559976e9bea8aa"
+http_archive(
+   name = "io_bazel_rules_kotlin",
+   urls = ["https://github.com/oppia/rules_kotlin/archive/%s.zip" % RULES_KOTLIN_VERSION],
+   type = "zip",
+   strip_prefix = "rules_kotlin-%s" % RULES_KOTLIN_VERSION,
+   sha256 = RULES_KOTLIN_SHA,
+)
+# TODO: Remove kt_download_local_dev_dependencies() when switching to rules_kotlin release
```

Comment:
Ditto for this TODO. Please file an issue for it and format it consistently.

Reply:
Done!